server/chat-commands/info.ts

```diff
@@ -738,6 +738,9 @@ export const commands: ChatCommands = {
 						allAdjacent: "All Adjacent Pok\u00e9mon",
 						any: "Any Pok\u00e9mon",
 						all: "All Pok\u00e9mon",
+						scripted: "Opposing Pok\u00e9mon that last dealt damage to the user",
```

Comment:
I'm not sure if we want to use this phrasing? I chose the name "scripted" because my plan was to use it for any target that the game chose for you.

Reply:
so would something like "Chosen Automatically" be better?

Reply:
Yeah, I like that.

Reply:
Done.